doc/source/user_guide/io.rst

```diff
@@ -170,10 +170,9 @@ dtype : Type name or dict of column -> type, default ``None``
      the default determines the dtype of the columns which are not explicitly
      listed.
 
-use_nullable_dtypes : bool = False
-    Whether or not to use nullable dtypes as default when reading data. If
-    set to True, nullable dtypes are used for all dtypes that have a nullable
-    implementation, even if no nulls are present.
+dtype_backend : {"numpy_nullable", "pyarrow"}, defaults to NumPy backed DataFrames.
+    Which dtype backend to use. If
+    set to True, nullable dtypes or pyarrow dtypes are used for all dtypes.
```

Comment:
Is True also an option here?

Reply:
Good catch, this shouldn't accept `True`

Reply:
Thx, updated